Grbl_Esp32/src/Config.h

```diff
@@ -233,7 +233,7 @@ Some features should not be changed. See notes below.
 // Number of homing cycles performed after when the machine initially jogs to limit switches.
 // This help in preventing overshoot and should improve repeatability. This value should be one or
 // greater.
-#define N_HOMING_LOCATE_CYCLE 1  // Integer (1-128)
+static const int NHomingLocateCycle=1;  // Integer (1-128)
```

Comment:
I think this needs a clang-format, I'm missing a space before/after the '='?

Reply:
I will run everything through clang-format en-masse.  I was doing it on a block by block basis in EMACS and I probably forgot to do that one-liner.